### ISSUE TITLE
Failed to locate takeover image

### ISSUE THREAD

**Issue by SlimeVagrant** [yellows8/oot3dhax#136]
I'm trying to reinstall OoT hax via powersaves for the first time in quite a while. I got it working way back when but now when I install the latest  version I get an error "Failed to locate takeover image". Not sure what I'm doing wrong.

**Comment by processor286** [comment]
I have this problem also. I had been using oothax some time ago, but now trying to get it working with 11.6.0.39-E and I get this.

I'm sure I've done this correctly, but hey maybe not.

Did you ever get your problem solved?

**Comment by SlimeVagrant** [comment]
Sure didn't. I don't think I'll ever get to play Theatrhytm Dragon Quest again unless I buy a japanese machine

**Comment by processor286** [comment]
You know what, I kicked myself. Leaving this public to maybe help others. When I was looking through the payloads to copy to PowerSaves, I scrolled to the bottom of the list and looked at all the matching numbers, yes. Was sure it was the correct one. Problem is, the bottom of the list is the POST5 versions, and I needed the  N3DS version. Of course :(

So, when I PowerSave copied the correct N3DS_E_27649_9221.bin, then it worked first time.

So maybe make sure you're using the N3DS file, and not the POST5 file. 

Hope you get it sorted anyway.
